src/modules/cloudflare/turnstile/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface ITurnstileVerificationResponse {
```

Comment:
I believe we should not use `snake_case` and `kebab-case` in JS/TS code at the same time, let's use `camelCase` instead

Reply:
Good point! camelCase and in same cases Pascal Case.

Reply:
The use of type follows return of cloudflare about turnstile, With successfully or not, will return how was structured on type.